doc/source/whatsnew/v2.0.0.rst

```diff
@@ -185,6 +185,7 @@ Performance improvements
 - Performance improvement in :func:`merge` and :meth:`DataFrame.join` when joining on a sorted :class:`MultiIndex` (:issue:`48504`)
 - Performance improvement in :meth:`DataFrame.loc` and :meth:`Series.loc` for tuple-based indexing of a :class:`MultiIndex` (:issue:`48384`)
 - Performance improvement for :meth:`MultiIndex.unique` (:issue:`48335`)
+- Reduce memory usage of DataFrame/Series `to_pickle` when using BZ2 or LZMA (:issue:`49068`)
```

Comment:
Sorry for all the nitpicks, can you please use the patterns used elsewhere:
:meth:`DataFrame.to_pickle` and  :meth:`Series.to_pickle`

Reply:
```suggestion
- Reduce memory usage of :meth:`DataFrame.to_pickle`/:meth:`Series.to_pickle` when using BZ2 or LZMA (:issue:`49068`)
```